LiteFarmOrg/LiteFarm
# LF-2172 UI changes in crop variety page

To test:
 1. create a crop variety by selecting any crop from the crop catalog page. (http://localhost:3000/crop_catalogue)
 2. add more varieties for the same crop. (around 5)  
 3. add a plan to any of the crop varieties.
 4. check for the count of active, planned, past, and flag of 'needs plan' ie. '!' in the red box at the top of the crop card.   

For more details: 
https://lite-farm.atlassian.net/browse/LF-2172
